node-project/programs/calculateFrequency.js

```diff
@@ -1,13 +1,15 @@
-exports.calculateFrequency = function (string) {
-  var frequency={};
-  var loop;
-  for(loop=0;loop<string.length;loop++)
-    {
-      var stringIterator=string.charAt(loop);
-      if(stringIterator.match(".*[a-z].*")){
-      if(frequency[string.charAt(loop)] ) {frequency[string.charAt(loop)]++;}
-      else {frequency[string.charAt(loop)]=1;}
-                                            }
+exports.calculateFrequency = function(string) {
+    var frequency = {};
+    var loop;
+    for (loop = 0; loop < string.length; loop++) {
+        var stringIterator = string.charAt(loop);
+        if (stringIterator.match(".*[a-z].*")) {
+            if (frequency[string.charAt(loop)]) {
```

Comment:
@psinha20 Assign 'string.charAt(loop)' in separate variable. So you don't need to call string.charAt function multiple times.